src/afl-fuzz-state.c

```diff
@@ -164,229 +164,227 @@ void afl_state_init(afl_state_t *afl, uint32_t map_size) {
 
 void read_afl_environment(afl_state_t *afl, char **envp) {
 
-  int   index = 0, issue_detected = 0;
+  int index = 0, issue_detected = 0;
   char *env;
   while ((env = envp[index++]) != NULL) {
 
-    if (strncmp(env, "ALF_", 4) == 0) {
+    if (strncmp(env, "ALF_", 4) != 0) {
```

Comment:
you inverse this. why?

Reply:
void read_afl_environment(afl_state_t *afl, char **envp) {

  int   index = 0, issue_detected = 0;
  char *env;
  while ((env = envp[index++]) != NULL) {

    if (strncmp(env, "ALF_", 4) == 0) {

      WARNF("Potentially mistyped AFL environment variable: %s", env);
      issue_detected = 1;

    } else if (strncmp(env, "AFL_", 4) == 0) {


I have a doubt， what is the meaning of the same condition as if and else branch？

Reply:
look closely, it is not the same. one is aLf, the other aFl :)

Reply:
> look closely, it is not the same
Ok, thanks for your reply